src/Bindings/ManualBindings_BlockArea.cpp

```diff
@@ -48,14 +48,12 @@ static int readCuboidOverloadParams(cLuaState & a_LuaState, int a_StartParam, cC
 	else
 	{
 		// Assume the 2-Vector3i version:
-		Vector3i * p1;
-		Vector3i * p2;
-		if (!a_LuaState.GetStackValues(a_StartParam, p1, p2))
+		Vector3i p1;
+		Vector3i p2;
```

Comment:
These are no longer used?

Reply:
No, just temporaries

Reply:
You use `a_Cuboid.p1` directly now, these variables are never used.